init.fish

```diff
@@ -3,7 +3,14 @@ function init -a path --on-event init_nvm
 
   if available fenv
     set -q NVM_DIR; or set -gx NVM_DIR ~/.nvm
-    fenv source $NVM_DIR/nvm.sh >/dev/null ^&1
+
+    if available brew; and brew --prefix nvm > /dev/null
+      set nvm_runnable (brew --prefix nvm)
+    else
+      set nvm_runnable $NVM_DIR
```

Comment:
I think that prefix isn't needed to be exported. Just making it global looks sufficient.